buildSrc/src/main/kotlin/java-conventions.gradle.kts

```diff
@@ -57,7 +57,7 @@ tasks {
     }
     jar {
         manifest {
-            attributes("Automatic-Module-Name" to "org.opentest4j.reporting.${project.name}")
+            attributes("Automatic-Module-Name" to "org.opentest4j.reporting.${project.name.replace('-', '.')}")
```

Comment:
Mh, did we ship `otr` with an illegal automatic module name? Good catch!

Reply:
Second thought: maybe `org.opentest4j.reporting` would be better? 🤔

Reply:
No, we didn't have any illegal names before. I've renamed `tooling` to `tooling-core` in this PR so this became necessary.